doc-sites/package-lock.json

```diff
@@ -0,0 +1,1876 @@
+{
```

Comment:
Do we want to commit this lock file?

Reply:
Yes. Lock files mean the versions are fixed and you don't get surprises when you npm install